src/Classes/TreeTab.lua

```diff
@@ -424,13 +459,26 @@ function TreeTabClass:OpenImportPopup()
 					if errMsg then
 						controls.msg.label = "^1"..errMsg
 						controls.import.enabled = true
+						return
 					else
-						decodeTreeLink(treeLink)
+						decodeTreeLink(treeLink, validateTreeVersion(treeLink:match(versionLookup)))
 					end
 				end)
 			end
+		elseif treeLink:match("poeskilltree.com/") then
+			local oldStyleVersionLookup = "/%?v=([0-9]+)%.([0-9]+)%.([0-9]+)#"
+			urlStyle = treeLink:match(oldStyleVersionLookup)
+			if urlStyle then
+				-- older style
+				-- Strip the version from the tree : https://poeskilltree.com/?v=3.6.0#AAAABAMAABEtfIOFMo6-ksHfsOvu -> https://poeskilltree.com/AAAABAMAABEtfIOFMo6-ksHfsOvu
+				decodeTreeLink(treeLink:gsub("/%?v=.+#","/"), validateTreeVersion(treeLink:match(oldStyleVersionLookup)))
+			else
+				decodeTreeLink(treeLink, validateTreeVersion(treeLink:match(versionLookup)))
+			end
```

Comment:
Sounds like the developer of poeskilltree isn't going to use the newer style, so this branch won't ever be hit.

Reply:
that was mentioned in the discord discussion.
Should we leave it there for future proofing ? in case of a change of heart ?